documentation/binding-data/tutorial-flow-components-binder-validation.asciidoc

```diff
@@ -82,13 +90,15 @@ binder.forField(nameField)
   .bind(Person::getName, Person::setName);
 ----
 
-The `withStatusLabel(Label label)` method sets the given label to show an error message if the validation fails.
-Instead of using labels you can set a custom validation status handler by the `withValidationStatusHandler` method.
-It allows customizing the way a binder displays error messages to get more flexibility than what
-`AbstractComponent.setComponentError(com.vaadin.server.ErrorMessage)` provides (it replaces the default behavior).
+* The `withStatusLabel(Label label)` method sets the given label to show an error message if the validation fails.
+
+As an alternative to using labels, you can set a custom validation status handler, using the `withValidationStatusHandler` method. This allows you to customize how the binder displays error messages and is more flexible than `AbstractComponent.setComponentError(com.vaadin.server.ErrorMessage)`.
```

Comment:
AbstractComponent.setComponentError is only in pre Vaadin 10. I'll change this to something valid.